### DISCUSSION TITLE
Custom Bootstrap themes

### DISCUSSION THREAD

**Discussion by renaudpawlak** [cincheo/daquota#37] [Announcements]
In addition to builtin Bootswatch themes, it is now very easy to use a custom theme from an external Bootstrap 4 CSS theme URL.

In the "Themes" menu, just fill in the wanted URL and (un)check the 'dark mode' option depending on whether the theme uses a dark background or not.

For instance, check out this nice Open Source CSS theme: https://startbootstrap.github.io/startbootstrap-sb-admin-2/css/sb-admin-2.min.css (you need to uncheck the dark mode option)

External web fonts should be supported  soon... ;)

**Comment by hasen666** [comment]
i was waiting for this ! many thanks !
